packages/components/src/components/text-field-wrapper/text-field-wrapper-styles.ts

```diff
@@ -63,6 +69,15 @@ export const getComponentCss = (
           ...(isNumber && {
             MozAppearance: 'textfield', // hides up/down spin button for Firefox
           }),
+          ...((isCalendar || isTime) && {
+            // for native placeholder color in safari, background ahs to be a special value, `transparent` or `rgba(0,0,0,0)` won't work
+            // this works nice for `theme="light"`, but for `theme="dark"` placeholder appears blue which is still better
+            // than having invisible dots or colons for data/time or not seeing the value at all after selection
+            '@supports (-webkit-hyphens: none)': {
+              background: 'rgba(0,0,1,0)',
+              color: isThemeDark(theme) ? getThemedColors(theme).primaryColor : 'initial',
```

Comment:
Why can't we set `color: "initial"` always? I thought `theme=dark` has no effect in Safari anyways?

Reply:
See native placeholders at the top with black `.` and `:` characters. These won't be visible on black background, only placeholder is. Once a value is chosen/entered the value will disappear as well since it is black by default.

<img width="706" alt="Bildschirmfoto 2024-03-05 um 08 30 40" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/f7d5500d-8f1d-41c5-8d2a-6ed4926bf7fb">